fedimintd/src/ui/configgen.rs

```diff
@@ -157,6 +157,7 @@ fn trusted_dealer_gen(
                     .iter()
                     .map(|(name, cfgs)| (name.to_string(), cfgs.0[&id].clone()))
                     .collect(),
+                max_connections: 1000,
```

Comment:
I see this number hard-coded in a bunch of places, should probably be a `const`.

Reply:
I'll make it a constant for the trusted and distributed configgen.  For the UI, I think it should be removed once the new branch is merged that calls distributed configgen directly.

Reply:
I really need to PR that ...